acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/das/DasCheckpointSyncAcceptanceTest.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Consensys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.test.acceptance.das;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.common.io.Resources;
+import java.io.IOException;
+import java.util.Optional;
+import java.util.stream.IntStream;
+import org.junit.jupiter.api.Test;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.spec.SpecMilestone;
+import tech.pegasys.teku.spec.datastructures.blocks.SignedBeaconBlock;
+import tech.pegasys.teku.test.acceptance.dsl.AcceptanceTestBase;
+import tech.pegasys.teku.test.acceptance.dsl.TekuBeaconNode;
+import tech.pegasys.teku.test.acceptance.dsl.TekuNodeConfigBuilder;
+
+public class DasCheckpointSyncAcceptanceTest extends AcceptanceTestBase {
+  @Test
+  public void shouldBeAbleToCheckpointSyncAndBackfillCustody() throws Exception {
+    // supernode with validators
+    final TekuBeaconNode primaryNode =
+        createTekuBeaconNode(
+            createConfigBuilder()
+                .withRealNetwork()
+                .withSubscribeAllCustodySubnetsEnabled()
+                // we don't want to make this test extreme, withhold once and don't repeat
+                .withInteropValidators(0, 64)
+                .build());
+
+    primaryNode.start();
+    final UInt64 genesisTime = primaryNode.getGenesisTime();
+    primaryNode.waitForNewFinalization();
+    final SignedBeaconBlock checkpointFinalizedBlock = primaryNode.getFinalizedBlock();
```

Comment:
I'd check this finalized block to NOT be the first fulu block, so that the secondary node will do a backfill. 
Ok so maybe since fulu activates on epoch 1, the wait for finalization will guarantee this. If true, I'd put this in a comment.